coral-hive/build.gradle

```diff
@@ -17,6 +17,7 @@ dependencies {
     exclude group: 'org.apache.avro', module: 'avro-tools'
   }
   compile deps.'hadoop'.'hadoop-common'
+  compile 'org.apache.ivy:ivy:2.4.0'
```

Comment:
Please add this to gradle/dependencies.gradle

Reply:
Done. I was planning to do this along with the other comment, and rebasing as follow-ups, but somehow the comments came too soon :)